lib/cocina/generator/generator.rb

```diff
@@ -78,7 +78,6 @@ def generate_for(schema)
         'rights_description_builder.rb',
         'title_builder.rb',
         'validatable.rb',
-        'validator.rb',
```

Comment:
Presumably we don't need to add the new validators to this list because they're in their own subdirectory?

Reply:
As it turns out, yes.